modules/geo-layers/src/tile-layer/tile-layer.js

```diff
@@ -27,14 +27,14 @@ const defaultProps = {
   // Use load directly so we don't use ResourceManager
   fetch: {
     type: 'function',
-    value: (url, {layer, signal}) => {
-      const loadOptions = {...layer.getLoadOptions()};
+    value: (url, {layer, loaders, options, signal}) => {
+      const loadOptions = {...options, ...layer.getLoadOptions()};
```

Comment:
This will overwrite the already merged `mvt` options. Should be `options || layer.getLoadOptions()`?

Reply:
Done